activerecord/lib/active_record/connection_adapters/postgresql/schema_statements.rb

```diff
@@ -589,6 +590,43 @@ def create_schema_dumper(options) # :nodoc:
           PostgreSQL::SchemaDumper.create(self, options)
         end
 
+        # Validates the given constraint.
+        #
+        # Validates the constraint named +constraint_name+ on +accounts+.
+        #
+        #   validate_foreign_key :accounts, :constraint_name
```

Comment:
This comment doesn't match the method it's documenting.

Reply:
It was fixed by 70c96b4423abacecb1116184c8ea4d4662b809f8